apigw-custom-domain/README.md

```diff
@@ -1,70 +1,69 @@
-# LocalStack Demo: API Gateway with Custom Domains
+# API Gateway with Custom Domains
 
-Simple demo application illustrating API Gateway (v2) endpoints using custom domain names (via Route53, ACM), deployed locally in LocalStack using the Serverless framework.
+[![Makefile CI](https://github.com/localstack-samples/localstack-pro-samples/actions/workflows/makefile.yml/badge.svg)](https://github.com/localstack-samples/localstack-pro-samples/actions/workflows/makefile.yml)
+
+| Key          | Value                             |
+| ------------ | --------------------------------- |
+| Environment  | LocalStack                        |
+| Services     | API Gateway, Lambda, Route53, ACM |
+| Integrations | Serverless Framework              |
+| Categories   | Serverless; REST API              |
+| Level        | Beginner                          |
+
+## Introduction
+
+A demo application showcasing API Gateway (v2) endpoints with custom domain names configured through Route53 and ACM, deployed locally using LocalStack and the Serverless framework. For more details, refer to the [documentation](https://docs.localstack.cloud/user-guide/aws/apigateway/#custom-domain-names-with-api-gateway).
 
 ## Prerequisites
 
-* LocalStack
+* [`localstack` CLI](https://docs.localstack.cloud/getting-started/installation/#localstack-cli) with [`LOCALSTACK_AUTH_TOKEN`](https://docs.localstack.cloud/getting-started/auth-token/)
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) with [`awslocal` wrapper script](https://docs.localstack.cloud/user-guide/integrations/aws-cli/#localstack-aws-cli-awslocal)
+* [Node.js 18.x](https://nodejs.org/en/download/package-manager) with `npm`
+* [Serverless Framework](https://www.serverless.com/framework/docs/getting-started) 3.x
 * Docker
-* Node.js / `npm`
-* `make`
+* `openssl` & `make`
 
-## Installing
+You can run `make check` to verify that all dependencies are installed.
 
-To install the dependencies:
-```
+## Installation
+
+Run the following command to install the necessary dependencies:
+
+```bash
 make install
 ```
 
-## Running
+## Start LocalStack
 
-Make sure that LocalStack is started:
-```
```

Comment:
I find all this verbosity in a sample's README a bit too much. Ideally, the install/run/check/whatever makefile targets are described in the root README, and in each sample's README we only describe what the application does.

We should strive to keep the use of words down to a minimum. And READMEs can easily become too convoluted, add too much noise, by repeating themselves.

I believe we need proper standardization across all sample apps in this repo. I would keep prerequisites, instructions, and whatever else stuff to the root README of this project. Everything else can fall into each app's README.

Also, since this repo is called `localstack-pro-samples`, it presumably only applies to pro samples, therefore the auth token is always needed. Hence, I would enforce the auth token env var to always be set when running the makefile target(s).